src/js/_store/modules/categories.js

```diff
@@ -83,5 +98,22 @@ export default {
     clearMessage({ commit }) {
       commit('clearMessage');
     },
+    confirmDeleteCategory({ commit }, { id, name }) {
+      const payload = { id, name };
+      commit('confirmDeleteCategory', payload);
```

Comment:
[nits]

以下のようにすればシンプルになるかと思います。

```suggestion
    confirmDeleteCategory({ commit }, payload) {
      commit('confirmDeleteCategory', payload);
```

Reply:
なるほど!ありがとうございます!